examples/confetti/src/main/scala/com/example/confetti/Confetti.scala

```diff
@@ -63,32 +73,39 @@ object Confetti extends IndigoDemo[Unit, Unit, Model, Unit]:
       .withFontSize(Pixels(12))
       .withColor(RGBA.White)
 
-  val helpText: TextBox =
-    TextBox("Click anywhere!", 640, 20).alignRight
+  def helpText(maybeButton: Option[MouseButton]): TextBox =
+    val msg = maybeButton.fold("Click anywhere!")(btn => s"$btn clicked!")
```

Comment:
"Click left and right mouse anywhere!" ?

Reply:
Your `(btn => s"$btn clicked!")` is bearly perceptible because it only flashes up for a moment.